### PR TITLE
ログアウトボタンの実装.ログアウト時に遷移する画面の変更.ログイン状態でログインボタンを非表示にしました

### PR DESCRIPTION
**Pull request by tkeshun**
#41

### CODE REVIEW

File: src/main/java/com/example/controller/LoginUserController.java [hunk 1]
```diff
@@ -24,42 +24,42 @@ public class LoginUserController {
...
+
 	@Autowired
 	private HttpSession session;
-	
+
 	/**
 	 * ログイン画面に遷移します.
 	 * 
-	 * @param model　モデル
+	 * @param model モデル
```
Comment (tkeshun) [inline src/main/java/com/example/controller/LoginUserController.java:34]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.RequireEmptyLineBeforeBlockTagGroupCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Javadoc tag '@param' should be preceded with an empty line.

Comment (tkeshun) [inline src/main/java/com/example/controller/LoginUserController.java:34]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/LoginUserController.java [hunk 2]
```diff
@@ -24,42 +24,42 @@ public class LoginUserController {
...
 	public String toLogin(Model model) {
 		return "ec/login";
 	}
-	
+
 	/**
 	 * ログインします.
 	 * 
-	 * @param form ログインフォーム
+	 * @param form  ログインフォーム
```
Comment (tkeshun) [inline src/main/java/com/example/controller/LoginUserController.java:45]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.RequireEmptyLineBeforeBlockTagGroupCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Javadoc tag '@param' should be preceded with an empty line.

Comment (tkeshun) [inline src/main/java/com/example/controller/LoginUserController.java:45]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/LoginUserController.java [hunk 3]
```diff
@@ -24,42 +24,42 @@ public class LoginUserController {
...
 	 * @param model モデル
 	 * @return 商品一覧画面
 	 */
 	@PostMapping("/login-user")
 	private String loginUser(LoginUserForm form, Model model) {
 		User user = loginUserService.loginUser(form.getEmail(), form.getPassword());
-		
-		if(user == null) {
+
+		if (user == null) {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/LoginUserController.java:53]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/LoginUserController.java:53]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/LoginUserController.java [hunk 4]
```diff
@@ -68,7 +68,7 @@ private String loginUser(LoginUserForm form, Model model) {
 	@GetMapping("/logout")
 	public String logout() {
 		session.invalidate();
-		return "redirect:/login-user/to-login";
+		return "redirect:/show-item/show-item-list";
```
Comment (tkeshun) [inline src/main/java/com/example/controller/LoginUserController.java:71]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/LoginUserController.java:71]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

**Review by uedamiku — APPROVED** [review]

**Pull request merged**